versioned_docs/version-1.0/language/crypto.mdx

```diff
@@ -159,8 +159,8 @@ struct PublicKey {
 }
 ```
 
-`PublicKey` supports two methods `verify` and `verifyPoP`.
-`verifyPoP` will be covered under [BLS multi-signature](#proof-of-possession-pop).
+`PublicKey` supports two methods `verify` and `verifyPoP`. 
+`verify` is the [signature verification](signature-verification) function, while `verifyPoP` is covered under [BLS multi-signature](#proof-of-possession-pop).
```

Comment:
CI flags this link is broken

Reply:
Fixed by [0e21bdf](https://github.com/onflow/cadence-lang.org/pull/116/commits/0e21bdfac8e60def837b34afb37ab780412c0dd5)